### ISSUE TITLE
Viewed file should be remembered 

### ISSUE THREAD

**Issue by filikos** [kasecato/vscode-intellij-idea-keybindings#237]
When following a code path down the codebase with CMD + Mouse I can navigate back the same way with ALT/Option + CMD + left arrow. 

If I have the same file opened two times my cursor suddenly navigates to the second opened file even though it was not in the view originaly.
This behavour can be avoided, when navigating "up" (ALT/Option + CMD + left arrow) or "down" (ALT/Option + CMD + right arrow) it would be more intuitive when the cursor remains in the file which was used to navigate originaly so the view will not suddenly change.

wdyt?

**Comment by kasecato** [comment]
I figured out that this was due to the default bindings in VSCode
So, I'll disable the default bindings of VS Code and perhaps we can resolve this

```json
// VS Code (Win)
{
  "key": "ctrl+alt+right",
  "command": "workbench.action.moveEditorToNextGroup"
}
// IntelliJ (Win)
{
  "key": "ctrl+alt+right",
  "command": "workbench.action.navigateForward",
  "when": "canNavigateForward"
}
```

```json
// VS Code (Win)
{
  "key": "ctrl+alt+left",
  "command": "workbench.action.moveEditorToPreviousGroup"
}
// IntelliJ (Win)
{
  "key": "ctrl+alt+left",
  "command": "workbench.action.navigateBack",
  "when": "canNavigateBack"
}
```

**Comment by kasecato** [comment]
Released v1.6.1
